src/Pages/Partenaires.js

```diff
@@ -21,22 +22,52 @@ class Partenaires extends Component {
           <meta name="description" content="Venez participez à la nationale nord-ouest 2020 de course d'orientation dans la région de Fontainebleau(77). L'événement propose 3 courses sur 3 jours (21, 22 et 23 août 2020)"/>
         </Helmet>
         <StyledSection>
-          <StyledH1HeaderTitle><Translator id="Partners.title" /></StyledH1HeaderTitle>
-          <StyledPartenairesLink href="https://www.onf.fr/">
-            <StyledImg src={onfLogo} alt={onfLogo}/>
-          </StyledPartenairesLink>
-          <StyledPartenairesLink href="http://www.ffcorientation.fr" paddingless hoverless>
-            <StyledImg src={ffcoLogo} alt={ffcoLogo}/>
-          </StyledPartenairesLink>
-          <StyledPartenairesLink href="https://www.iledefrance.fr/">
-            <StyledImg src={iDFLogo} alt={iDFLogo}/>
-          </StyledPartenairesLink>
-          <StyledPartenairesLink href="https://www.lifco.fr/">
-            <StyledImg src={ligueIDFLogo} alt={ligueIDFLogo}/>
-          </StyledPartenairesLink>
-          <StyledPartenairesLink href="http://cd77if.free.fr/">
-            <StyledImg src={cDCO77Logo} alt={cDCO77Logo}/>
-          </StyledPartenairesLink>
+          <StyledH1><Translator id="Partners.title" /></StyledH1>
+          <StyledBorderBox>
```

Comment:
Je crois que ici les images sur mobile vont sortir de l'écran c'est pour ça que j'qvqis mis un component supplémentaires

Reply:
Yes I fixed it now